### PR TITLE
EWPP-607: Remove unused preprocess from language selection page.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

**Review by 22Alexandra — APPROVED** [review]

File: modules/oe_multilingual_selection_page/oe_multilingual_selection_page.module [hunk 1]
```diff
@@ -6,22 +6,3 @@
  */
 
 declare(strict_types = 1);
-
-/**
- * Implements hook_page_preprocess().
- */
-function oe_multilingual_selection_page_preprocess_language_selection_page_content(array &$variables): void {
```
Comment (upchuk) [inline modules/oe_multilingual_selection_page/oe_multilingual_selection_page.module:13]: Why did you remove this? What was this used for? And how would it break a site that uses oe_multilingual and now doesn't have this code here?

**Review by upchuk — CHANGES_REQUESTED** [review]

File: modules/oe_multilingual_selection_page/oe_multilingual_selection_page.module (hunk 1 above)
Comment (nagyad) [reply modules/oe_multilingual_selection_page/oe_multilingual_selection_page.module:13]: This preprocess was used for ecl-language list back in the days. It does not serve anything to the contrib module: language_selection_page I checked. This is a left over after a refactor we did in oe_theme where we changed the ecl-language-list since then.

Comment (upchuk) [reply modules/oe_multilingual_selection_page/oe_multilingual_selection_page.module:13]: It's very difficult to predict what site is relying on this. I get that we dont use it anymore, and 99% others dont either but if somebody does rely on this preprocessing, they are gonna have a bad surprise. Put a comment above saying this is deprecated and can be removed in 3.x or something.

**Review by upchuk — CHANGES_REQUESTED** [review]

**Comment by ademarco** [comment]
Yeah I agree with @upchuk here, even though we used those HTML properties in an old version of ECL, the two things are not necessarily connected. I'd simply remove any reference to ECL and keep it there as it is, since these HTML attributes can be useful anyway.

**Comment by nagyad** [comment]
Closing PR after assessing that the removal of the code would pose potential issue to other sites that rely on this code.
